process-migration-service/src/main/resources/project-defaults.yml

```diff
@@ -0,0 +1,23 @@
+swarm:
+  deployment:
+    process-migration.war:
+      jaxrs:
+        application-path: /
+  logging:
+    org.kie.processmigration:
+      level: INFO
+  datasources:
+    data-sources:
+      ExampleDS:
+        driver-name: mysql
+#add the ?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC to avoid timezone related exception        
+        connection-url: jdbc:mysql://localhost:3306/rhpam?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
```

Comment:
any reason why mysql is default? in most (if not all) cases we use H2 as default to it can be tested as part of builds

Reply:
No reason at all, we just wanted to provide an example of use for mysql. For H2 there's the test-defaults.yml. In any case a user would want to provide a completely different configuration file. Maybe I should remove this file completely or move it to the test folder with more examples.